src/pint/pint_matrix.py

```diff
@@ -72,16 +79,65 @@ def label_units(self):
             units.append(self.get_axis_labels(dim))
         return units
 
-    def get_label_size(self, label):
-        """Get the size of the a label in each axis.
+    def get_label_names(self, axis=None):
+        """return only the names of the labels
```

Comment:
Could you please put this in docstring format? First line is a short summary, in the form of a complete sentence with capital letter and period. This will show up correctly in the documentation.